apps/wallets/smart-wallet/next/src/components/header.tsx

```diff
@@ -6,101 +6,104 @@ import { Copy, Image as ImageIcon, User, WalletMinimal } from "lucide-react";
 import Link from "next/link";
 import { useRouter } from "next/navigation";
 
-import { useAuth } from "@crossmint/client-sdk-react-ui";
+import { EVMSmartWallet, useAuth, useWallet } from "@crossmint/client-sdk-react-ui";
 
 import { Avatar, AvatarFallback, AvatarImage } from "./avatar";
 import { DropdownMenu, DropdownMenuContent, DropdownMenuTrigger } from "./dropdown-menu";
 import { Typography } from "./typography";
 import { useToast } from "./use-toast";
 
-export const Header = () => {
-    const { jwt, logout, wallet } = useAuth();
+function formatWalletAddress(address: string, startLength: number, endLength: number): string {
+    return `${address.substring(0, startLength)}...${address.substring(address.length - endLength)}`;
+}
+
+export const Header: React.FC = () => {
+    const { logout } = useAuth();
+    const { wallet, status: walletStatus } = useWallet();
```

Comment:
I rename status every time I use it, maybe that's a good sign to just rename status => walletStatus

cc @jmderby what do other libs normally do, wdyt?

Reply:
Are we using `status` from `useAuth` anywhere within the demo? If not, I think it's acceptable to keep the name status. My thinking is that in the uncommon case where useAuth and useWallet are being used in a component and the status of each is being used, it would be reasonable for the developer to rename the status at that point. Additionally, in scenarios where the wallet is loading and we're not relying on any status from the useAuth hook, using status seems appropriate since the page is primarily concerned with the wallet's status.

Reply:
ok sgtm